conda_spawn/activate.py

```diff
@@ -1077,6 +1077,12 @@ def _hook_postamble(self) -> str:
         return "Remove-Variable CondaModuleArgs"
 
 
+class ShellActivator(PosixActivator):
+    pathsep_join = ";".join if on_win else ":".join
```

Comment:
This is surprising 🤔 Isn't shell supposed to be cross-platform? Why do they need different separators 🤔 Just curious.